### ISSUE TITLE
The size of result is not what I want. Can you help me?

### ISSUE THREAD

**Issue by IsDling** [junyanz/pytorch-CycleGAN-and-pix2pix#214]
My data is aligned and 3504 * 2336,I use my dataset to train and get the result.The result image is 256 * 256 and only a part of the result what I want. How can I do to get  that (input image size 3504*2336 or others and output image size is same as input image size )? Thank you

**Comment by junyanz** [comment]
What are the `loadSize` and `fineSize` in your training and test scripts?

**Comment by IsDling** [comment]
loadSize 286 and fineSize 256,how can I change to get the result 3504*2336

**Comment by avijit9** [comment]
@IsDling  found any solution yet?

**Comment by IsDling** [comment]
@avijit9 I solve that with that add padding to make the image to 3504*3504(not resize!) And then, when train, set   loadSize 3504 and fineSize 256, when test ,crop the image in order to several 256*256 small image ,set loadSize 256 and fineSize 256,then combine the result in order. The **import** is your dataset must be aligned, result will not be  ideal if not. May help you!

**Comment by junyanz** [comment]
I am not sure if a 12GB GPU can fit 3000x2300 image during the test time. If you just want to work on 1k image, you can set loadSize=1024, fineSize=256 during training, and use loadSize=1024, fineSize=1024 during test.

**Comment by mrluker** [comment]
@junyanz @avijit9 @IsDling 

You can test a 2820x1880 image on a 8GB GPU so I'm confident you could do that on a 12GB GPU. I have gotten resolutions of at least 4800x3,200 on a P6000 with 24gb of ram

the trick is, for test.py use ```--model cycle_gan --dataset_mode unaligned``` instead of ```--model test --dataset_mode single```

using ```--model test``` appears to take more VRAM and thus can only run at a lower resolution

see issue: https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/236

**Comment by IsDling** [comment]
@mrluker  thank you I will try

**Comment by IsDling** [comment]
@liang233 firstly,you must make your data set aligned, and then you can crop your original images to some images which have smaller size. For example,if we have some a image which size is 400*400,then you can  crop it to 4 images with size 200*200.Train the model with smaller images and when test,you can use model to generate some images with size 200*200,and then merging them in order,so you can get a result with size 400*400.But it have some disadvantage that the result may lost some texture information. You can try this method with your data set.Good luck!

**Comment by liang233** [comment]
thank you,I try to padding my images because the largest size of my images is 1000,and the loss of texture information is  unacceptable now
